nomad/state/state_store.go

```diff
@@ -1972,21 +1864,13 @@ func (s *StateStore) CSIPluginByID(ws memdb.WatchSet, id string) (*structs.CSIPl
 	return plug, nil
 }
 
-// CSIPluginDenormalize returns a CSIPlugin with jobs
+// CSIPluginDenormalize returns a CSIPlugin with allocation details
 func (s *StateStore) CSIPluginDenormalize(ws memdb.WatchSet, plug *structs.CSIPlugin) (*structs.CSIPlugin, error) {
 	if plug == nil {
 		return nil, nil
 	}
 
-	for ns, js := range plug.Jobs {
-		for id := range js {
-			j, err := s.JobByID(ws, ns, id)
-			if err != nil {
-				return nil, err
-			}
-			plug.Jobs[ns][id] = j
-		}
-	}
+	// FIXME do we need allocation details?
```

Comment:
It seems like if we never do, then this method doesn't do anything?

Reply:
ugh, that's ridiculous. The FIXME was there so that I would definitely not forget to implement that method. It has now been implemented.